### PR TITLE
take into account listing fee and guild cut

### PR DESCRIPTION
**Pull request by marcusklaas**
The profit you see now should be the money you expect to receive after taxes and listing fees.

### CODE REVIEW

**Comment by marcusklaas** [comment]
It's probably nicer to alter the profit instead of the actual pricePerPiece... Let me fix this real quick!

File: JMSuperDeal.lua [hunk 1]
```diff
@@ -377,21 +377,22 @@ end
...
     end
 
-    -- Sale is not expensive enough
-    if priceSuggestion.pricePerPiece <= item.pricePerPiece then
+    -- Because we buy 10 items so we get 10 times that profit if we buy this
+    local profit = (priceSuggestion.pricePerPiece * (100 - taxPercentage) / 100 - item.pricePerPiece) * item.stackCount 
+    local profitPercentage = ((profit / item.stackCount) / item.pricePerPiece) * 100
+    
+    -- Sale is not profitable
+    if profit < 0
```
Comment (JordyMoos) [inline JMSuperDeal.lua]:
``` lua
if profit < 0 then
```

Comment (marcusklaas) [inline JMSuperDeal.lua]: Ah, yup. 1 sec ;-)

**Comment by JordyMoos** [comment]
Nice useful addition to make the profit more precise :+1:

**Pull request merged**